models/stg_facebook_ads__creative_history.sql

```diff
@@ -36,7 +45,11 @@ final as (
         name as creative_name,
         page_link,
         template_page_link,
-        url_tags,
+        {% if ns.url_tags_column_type == 'json' -%}
```

Comment:
Should we consider adding a bigquery conditional here as well? I know there's a conditionally already where an is set, but wondering what the chances are that this is used on a non bigquery destination.